ansible_collections/arista/avd/docs/release-notes/3.x.x.md

```diff
@@ -680,9 +680,9 @@ Provides major enhancements and data model has been updated!
         - eos_designs.
         - eos_cli_config_gen.
 
-    Upgrade functionality has been added to the `eos_designs` role. See [README](../../roles/eos_designs/#upgrade-of-eos_designs-data-model) for details.
+    Upgrade functionality has been added to the `eos_designs` role. See [README](https://avd.sh/en/v3.7.0/roles/eos_designs/index.html#upgrade-of-eos_designs-data-model) for details.
```

Comment:
why 3.7.0?

Reply:
because that is the only thing we can link to until we release 3.8. Since the docs will disappear from devel when this pr merges. And stable will soonish become 4.0